### PR TITLE
[fx] add test for meta tensor.

### PR DESCRIPTION
**Pull request by super-dainiu**
I added test for meta tensor about its shape, dtype, and stride. Also, I checked the correctness of back-propagation.
image

### CODE REVIEW

**Review by Cypher30 — APPROVED** [review]
Great Work! In the future if we are able to retrieve the actual code with both forward and backward (and if this feature is needed by us), we could add some compatibility test that verify the forward & backward consistency.

**Review by Cypher30 — CHANGES_REQUESTED** [review]
Please address the CI failure~

File: tests/test_fx/test_meta/test_aten.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from colossalai.fx.profiler import MetaTensor
+
+import pytest
+
+try:
+    meta_lib = torch.library.Library("aten", "IMPL", "Meta")
+    incompatible = False    # version > 1.12.0
```
Comment (FrankLeeeee) [inline tests/test_fx/test_meta/test_aten.py:11]: Use constant for this value  i.e. `INCOMPATIBLE = False`

Comment (super-dainiu) [reply tests/test_fx/test_meta/test_aten.py]: Solved.

**Review by Cypher30 — APPROVED** [review]

**Pull request merged**